assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.test.js

```diff
@@ -59,12 +59,17 @@ const setupRegistryNoFieldDataDesktop = ( { dispatch } ) => {
 describe( 'DashboardPageSpeed', () => {
 	afterEach( fetchMock.mockClear );
 
-	it( 'renders a progress bar while reports are requested', () => {
+	it( 'renders a progress bar while reports are requested', async () => {
 		freezeFetch( /^\/google-site-kit\/v1\/modules\/pagespeed-insights\/data\/pagespeed/ );
 
 		const { queryByRole } = render( <DashboardPageSpeed />, { setupRegistry: setupRegistryNoReports } );
 
-		expect( queryByRole( 'progressbar' ) ).toBeInTheDocument();
+		await waitFor( () => {
+			expect( queryByRole( 'progressbar' ) ).toBeInTheDocument();
+		} );
+
+		expect( console ).toHaveErrored();
+		expect( console ).toHaveWarned();
```

Comment:
`waitFor` looks great here! Can you elaborate on why we would expect both an errors and warnings here? It isn't immediately obvious as to why we'd expect either since the previous test didn't.

Reply:
After working through this with Evan, this was fixed by adding another `freezeFetch` call as the endpoint is called twice (once for desktop, the other for mobile).